content/en/docs/contribute/localization.md

```diff
@@ -236,7 +236,7 @@ labels:
 - language/es
 ```
 
-After adding the language-specific `OWNERS` file, update the [root
+After adding the language-specific `OWNERS` file, update the root [
 `OWNERS_ALIASES`](https://git.k8s.io/website/OWNERS_ALIASES) file with the new Kubernetes teams
```

Comment:
I think it should be:
```markdown
After adding the language-specific `OWNERS` file, update the
[root `OWNERS_ALIASES`](https://git.k8s.io/website/OWNERS_ALIASES) file with the new Kubernetes teams
```

Reply:
Then I have to close this PR and make another new for this?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
If anyone clicked on [root `OWNERS_ALIASES`] it will redirect into `owners_aliases` docs and there root is not mentioned anywhere. So this `root` should be out of that link.